ahk-script/ibus-table-generator.py

```diff
@@ -0,0 +1,192 @@
+import os
+import uuid
+import datetime
+
+now = datetime.datetime.now().strftime("%Y%m%d")
+
+# taken and modified from here: https://github.com/neroist/sitelen-pona-ucsur-guide/blob/main/tokipona.txt
+# created by jan Komi and edited by authors of sitelen pona ucsur guide and kule Piton
```

Comment:
afaik jan Lili did not edit the table, majority of changes are from jan Kekan San and `@Qwerty-Space`. mi pali lili taso li pana e nimi su tawa ona